application/perfect/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM swift:5.2.5
+
+RUN apt-get update
+
+RUN apt-get install -y openssl libssl-dev uuid-dev
+
+RUN echo "deb [trusted=yes] https://apt.secrethub.io stable main" > /etc/apt/sources.list.d/secrethub.sources.list && apt-get update
+RUN apt-get install -y secrethub-cli
+
+RUN git clone https://github.com/PerfectlySoft/PerfectTemplate.git
+
+COPY  /PerfectTemplate/Sources/PerfectTemplate/main.swift /PerfectTemplate/Sources/PerfectTemplate/main.swift
```

```suggestion
COPY  ./PerfectTemplate/Sources/PerfectTemplate/main.swift /PerfectTemplate/Sources/PerfectTemplate/main.swift
```
I think we should use relative paths so that we make it clear that we're copying from the current directory.

---

Maybe you'll like this current version even better, it's shorter and achieves the same result.